server/game/StatModifier.ts

```diff
@@ -0,0 +1,65 @@
+import { CardTypes } from "./Constants";
+
+class StatModifier {
```

Comment:
I'm not really sure what this is for - is it just to make your stats display work nicely?

Reply:
Yeah this is just to make the generation of the modifiers a bit simpler in `drawcard.js`